web/tests/compose.test.js

```diff
@@ -166,6 +166,7 @@ test_ui("send_message", ({override, override_rewire, mock_template}) => {
     const $child = $(".unread_count");
     $container.set_find_results(".unread_count", $child);
 
+    compose_banner.clear_all = () => {};
```

Comment:
In response to your question on CZO --  this seems fine! It's also fine to mock out a function that calls `clear_all()` as long as it isn't involved in the main functionality you're testing. There are probably a few acceptable options for what you mock.